js/main.js

```diff
@@ -0,0 +1,153 @@
+//---------------------- Add to cart --------------------------------
+const addEventBtn = (products) => {
+  const btnOrder = document.querySelectorAll('.add-btn');
+  
+  console.log(btnOrder);
+
+  btnOrder.forEach(button => {
+    button.addEventListener('click', () => {
+      const productCategory = button.getAttribute('data-category');
+      const productId = button.getAttribute('data-id');
+
+      const product = products.find(item => item.id === +productId);
+      console.log(product);
+
+      const cartItems = JSON.parse(localStorage.getItem('cart'));
+
+      if(cartItems && cartItems.length > 0) {
+        const existedItem = cartItems.find(item => item.id === product.id && item.title === product.title);
+
+        console.log(existedItem);
+
+        if(existedItem) {
+          const updatedCart = cartItems.map(item => {
+            if(item.id === product.id && item.title === product.title) {
+              console.log(cartItems);
+              return {...item, amount: Number(item.amount) + 1}
+            }
+            return item;
+          });
+          console.log(updatedCart);
+          localStorage.setItem('cart', JSON.stringify(updatedCart));
+          const quantity = updatedCart.reduce((total, item) => total + item.amount, 0);
+          console.log(quantity);
+          localStorage.setItem('quantity', JSON.stringify(quantity));
+          document.querySelector('#cart-number').innerHTML = quantity;
+        } else {
+          cartItems.push({...product, amount: 1});
+          localStorage.setItem('cart', JSON.stringify(cartItems));
+          const quantity = cartItems.reduce((total, item) => total + item.amount, 0);
+          console.log(quantity);
+          localStorage.setItem('quantity', JSON.stringify(quantity));
+          document.querySelector('#cart-number').innerHTML = quantity;
+        }
+      } else {
+        localStorage.setItem('cart', JSON.stringify([{...product, amount: 1}]));
+        localStorage.setItem('quantity', JSON.stringify(1));
+        document.querySelector('#cart-number').innerHTML = 1; 
+      }
+    });
+  });
+};
+
+//---------------------- Delete from cart --------------------------------
+const deleteEventBtn = () => {
+  const btnDelete = document.querySelectorAll('.delete-btn');
+
+  btnDelete.forEach(button => {
+    button.addEventListener('click', () => {
```

Comment:
```suggestion
    button.addEventListener('click', removeProductFromCart)
```
Tách riêng 1 hàm luôn